src/messages.tsx

```diff
@@ -14,3 +14,5 @@ export const ValidationMessage = (filePath: string) => ({
   error: () => `File: ${filePath} does not exists or is not a file!`,
   message: () => `File: ${filePath} successfully validated!`
 });
+
+export const NO_SPEC_PATH_FOUND = 'No spec path found in your working directory, please use flags or store a context';
```

Comment:
We need to do better here. User, when running `asyncapi validate` most probably do not have an idea about all our options, defaults, and way we load files, priorities - we need to make it clear. Good help messages are a key for good developer experience.

how about? 👇🏼 

```suggestion
export const NO_SPEC_FOUND = 'Unable to perform validation. Specify what AsyncAPI file should be validated.\n\nThese are your options to specify in the CLI what AsyncAPI file should be used:\n- You can use --file flag to provide a path to the AsyncAPI file: asyncapi validate --file path/to/file/asyncapi.yml\n- You can use --context flag to provide a name of the context that points to your AsyncAPI file: asyncapi validate --context mycontext\n- In case you did not specify a context that you want to use, the CLI checks if there is a default context and uses it. To set default context run: asyncapi context use mycontext\n- In case you did not provide any reference to AsyncAPI file and there is no default context, the CLI detects if in your current working directory you have files like asyncapi.json, asyncapi.yaml, asyncapi.yml. Just rename your file accordingly.';
```

Now, above is just a suggestion of how message should be printed, but from code perspective it should be splitted

Part `Unable to perform validation. Specify what AsyncAPI file should be validated.` should be only in context of `asyncapi validation` and the rest of the message should be reusable. Why reusable? because it will be always used in other commands when the same error will happen that there is no file specified, when we will have command `asyncapi optimize` it will be the same. So you need to make sure that examples that I provided in the message are customizable, in context of validation user see `asyncapi validate --file path/to/file/asyncapi.yml`, but in context of optimization user will see `asyncapi optimize --file path/to/file/asyncapi.yml`